src/celluloid-application.c

```diff
@@ -307,6 +308,48 @@ options_handler(GApplication *gapp, GVariantDict *options, gpointer data)
 	return version?0:-1;
 }
 
+static gboolean
+local_cmdline_handler (	GApplication *gapp,
+			gchar ***arguments,
+			gint *exit_status)
+{
+	CelluloidApplication *app = CELLULOID_APPLICATION(gapp);
+	gchar **argv = *arguments;
+	g_clear_pointer(&app->mpv_options, g_free);
+
+	GArray *options = g_array_new(TRUE, FALSE, sizeof(gchar *));
+	gint i = 1;
+	while(argv[i])
+	{
+		if(g_str_has_prefix(argv[i], "--mpvopt-"))
+		{
+			gchar *option = g_strconcat("--", &argv[i][sizeof("--mpvopt-")-1], NULL);
```

Comment:
Might be a bit clearer like this:
```suggestion
			gchar *option = g_strconcat("--", argv[i] + sizeof("--mpvopt-") - 1, NULL);
```